cpuinfo_linux.go

```diff
@@ -158,3 +159,39 @@ func getCPUVariant() (string, error) {
 
 	return variant, nil
 }
+
+func getAmd64MicroArchLevel() (string, error) {
```

Comment:
https://github.com/tonistiigi/go-archvariant is an alternative here that does not depend on proc or Linux.